src/summary/profile/age-helpers.tsx

```diff
@@ -0,0 +1,62 @@
+// Based on http://www.gregoryschmidt.ca/writing/patient-age-display-ehr-conventions,
```

Comment:
It looks like you implemented "Partial Abbreviation no spaces" while the "? Preferred option ?" is "Partial abbreviation with spaces." This is an intentional departure from the text of the blog post?

Reply:
No just an oversight on my part. Will change

Reply:
updated